src/pages/signIn/index.tsx

```diff
@@ -0,0 +1,83 @@
+import { useTranslation } from 'react-i18next';
+import Button from 'src/modules/General/components/Button';
+import Icon from 'src/modules/General/components/Icon';
+import { Input } from 'src/modules/General/components/Input';
+import { Link } from 'src/modules/General/components/Link';
+
+import css from './index.module.scss';
+import { useSignIn } from './useSignIn';
+
+export const SignIn = () => {
+  const { register, errors, handleSubmit } = useSignIn();
+  const { t } = useTranslation();
+
+  return (
+    <div className={css['container']}>
+      <div className={`w-full lg:min-w-[600px] ${css['formSection']}`}>
+        <img src={'/images/logo/logo.svg'} alt="SHIN_LOGO" width="79px" height="32px" />
+        <div className={css['formSection__form']}>
+          <div className={`w-full lg:w-[360px] lg:min-w-[360px] ${css['formSection__form__content']}`}>
+            <div className={css['formSection__form__title']}>
+              <div className={css['formSection__form__h1']}>{t('h1')}</div>
+              <div className={css['formSection__form__h2']}>{t('h2')}</div>
```

Comment:
- `form__title`
- `form__h1`
- `form__h2`

in BEM mentiond shouldn;t be use `x__y__z` (prevent to using two or more `__`)